### PR TITLE
Doc update

### PR DESCRIPTION
**Pull request by johnkim76**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,163 @@
...
+LABEL "com.redhat.apb.version"="0.1.0"
+LABEL "com.redhat.apb.spec"="some-long-blob-value"
+
+ADD playbooks /opt/apb/actions
+ADD roles /opt/ansible/roles
+
+USER apb
+```
+
+#### Spec File (`apb.yml`)
```
Comment (fabianvf) [inline README.md]: Might want to include the full path to the files for these

File: README.md [hunk 2]
```diff
@@ -1,2 +1,163 @@
...
+│   └── deprovision.yml
+└── roles
+    └── my-apb-openshift
+        ├── defaults
+        │   └── main.yml
+        ├── tasks
+            └── main.yml
+```
+### Directory Structure with Optional Files
+The directory structure with `optional/additional` files, may look like this:
```
Comment (cfchase) [inline README.md]: nitpick: was there a reason this `optional/additional` looks like a code block?

**Comment by cfchase** [comment]
Minor nitpick issues, aside, it looks good to me.

ACK

File: README.md (hunk 1 above)
Comment (johnkim76) [reply README.md]: thanks.. will update

File: README.md (hunk 2 above)
Comment (johnkim76) [reply README.md]: thanks, will update

**Review by fabianvf — APPROVED** [review]

**Pull request merged**